tests/components/device_tracker/test_automatic.py

```diff
@@ -0,0 +1,254 @@
+                        "country": "US"
+                    },
+                    "path": "path",
+                    "vehicle_events": [],
+                    "start_timezone": "America/Denver",
+                    "end_timezone": "America/Denver",
+                    "idling_time_s": 0,
+                    "tags": []
+                },
+                {
+                    "url": "https://api.automatic.com/trip/tid2/",
+                    "id": "tid2",
+                    "driver": "https://api.automatic.com/user/uid/",
+                    "user": "https://api.automatic.com/user/uid/",
+                    "started_at": "2016-08-28T18:48:00.727000Z",
+                    "ended_at": "2016-08-28T18:55:25.800000Z",
+                    "distance_m": 3969.1,
+                    "duration_s": 445.1,
+                    "vehicle": "https://api.automatic.com/vehicle/vid/",
+                    "start_location": {
+                        "lat": 32.87336,
+                        "lon": -117.22743,
+                        "accuracy_m": 11
+                    },
+                    "start_address": {
+                        "name": "123 Fake St, Nowhere, NV, USA",
+                        "display_name": "Fake St, Nowhere, NV",
+                        "street_number": "123",
+                        "street_name": "Fake St",
+                        "city": "Nowhere",
+                        "state": "NV",
+                        "country": "US"
+                    },
+                    "end_location": {
+                        "lat": 32.82336,
+                        "lon": -117.23743,
+                        "accuracy_m": 10
+                    },
+                    "end_address": {
+                        "name": "321 Fake St, Nowhere, NV, USA",
+                        "display_name": "Fake St, Nowhere, NV",
+                        "street_number": "Unknown",
+                        "street_name": "Fake St",
+                        "city": "Nowhere",
+                        "state": "NV",
+                        "country": "US"
+                    },
+                    "path": "path",
+                    "vehicle_events": [],
+                    "start_timezone": "America/Denver",
+                    "end_timezone": "America/Denver",
+                    "idling_time_s": 0,
+                    "tags": []
+                }
+            ]
+        }, 200)
+    else:
+        _LOGGER.debug('UNKNOWN ROUTE')
+
+
+class TestAutomatic(unittest.TestCase):
+    """Test cases around the automatic device scanner."""
+
+    def see_mock(self, **kwargs):
+        """Mock see function."""
+        self.assertEqual('vid', kwargs.get('dev_id'))
+        self.assertEqual(FUEL_LEVEL,
+                         kwargs.get('attributes', {}).get('fuel_level'))
+        self.assertEqual((LATITUDE, LONGITUDE), kwargs.get('gps'))
+        self.assertEqual(ACCURACY, kwargs.get('gps_accuracy'))
+
+    def setUp(self):
```

Comment:
Yeah - get no errors for that line.

On Fri, Sep 2, 2016 at 3:28 PM, Johann Kellerman <notifications@github.com>
wrote:

> In tests/components/device_tracker/test_automatic.py
> <https://github.com/home-assistant/home-assistant/pull/3035#discussion_r77414282>
> :
>
> > +    else:
> > +        _LOGGER.debug('UNKNOWN ROUTE')
> > +
> > +
> > +class TestAutomatic(unittest.TestCase):
> > +    """Test cases around the automatic device scanner."""
> > +
> > +    def see_mock(self, **kwargs):
> > +        """Mock see function."""
> > +        self.assertEqual('vid', kwargs.get('dev_id'))
> > +        self.assertEqual(FUEL_LEVEL,
> > +                         kwargs.get('attributes', {}).get('fuel_level'))
> > +        self.assertEqual((LATITUDE, LONGITUDE), kwargs.get('gps'))
> > +        self.assertEqual(ACCURACY, kwargs.get('gps_accuracy'))
> > +
> > +    def setUp(self):
>
>     def setUp(self):  # pylint: disable=invalid-name
>
> Have you tried running script/lint --changed?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/home-assistant/home-assistant/pull/3035/files/25350234737ddf86503b9a62bdd51f5ed4445076#r77414282>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC2fZYqAzY6UFcVBlB-Cmf1Zn0qP7WXrks5qmJT8gaJpZM4JvIoC>
> .
>